### PR TITLE
fix(eslint): Addresses 'indent' violations

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: https://jira.rax.io/browse/FRMW-726

### LGTMs
- [x] Dev LGTM
- [x] Dev/QE LGTM

### CODE REVIEW

File: src/components/rxAccountInfo/docs/rxAccountInfo.js [hunk 1]
```diff
@@ -4,41 +4,37 @@
...
+           name: 'Attribution'
+       }, {
+           url: 'http://mirrors.creativecommons.org/presskit/icons/nc.large.png',
+           description: 'You may not use the material for commercial purposes.',
+           name: 'Non-Commercial'
+       }, {
+           url: 'http://mirrors.creativecommons.org/presskit/icons/zero.large.png',
+           description: 'Waives as many rights as legally possible, worldwide.',
+           name: 'Public Domain'
+       }]
```
Comment (CITguy) [inline src/components/rxAccountInfo/docs/rxAccountInfo.js]: Is it valid to have the array brackets on different lines than the object brackets?

```javascript
.value('Badges', [
    {
        /* ... */
    }, {
        /* ... */
    }
])
```

Comment (CITguy) [inline src/components/rxAccountInfo/docs/rxAccountInfo.js]: I feel like this is a pattern I see more often than what is proposed in the change.

Comment (CITguy) [inline src/components/rxAccountInfo/docs/rxAccountInfo.js]: I have comments to add to remaining changes, but I'll hold off until we chat about this one first.

**Comment by CITguy** [comment]
There are fewer indentation issues than I'd expected. :smile:

**Comment by parlarjb** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**